src/api.py

```diff
@@ -2,8 +2,10 @@
 from sanic.response import json
 from sanic_jwt.decorators import inject_user, protected
 from sanic_jwt.exceptions import AuthenticationFailed
-
 from src.utils import expects_json_object
+from config import client, secret, redirect_uri
+from src.services import UserService
+import requests
```

Comment:
I think we can add `requests` to Poetry as well

Reply:
Isnt this module inside alr?

Reply:
Yes as a dependency of one of our packages. But if we are going to use it directly, it is better to add it to Poetry so that we can pin the version and whatnot